.github/workflows/macos.yml

```diff
@@ -24,3 +24,28 @@ jobs:
     - name: Build Manual makefile
       working-directory: mac
       run: make -f Makefile-manual
+
+  build-cmake:
+
+    runs-on: macos-latest
```

Comment:
Use GitHub Actions' matrix feature to avoid having separate jobs for each OS. For example: https://github.com/Be-ing/portaudio/blob/3b2bf31b447c74919ded00fd0552e566dcada171/.github/workflows/c-cpp.yml

Reply:
thanks, I'll look into it

Reply:
btw, does the matrix variances run in parallel?

Reply:
Yup! And GitHub Actions is very generous with computing resources. You get 20 concurrent jobs for free (max 5 macOS jobs).